migrations/2_deploy_contracts.js

```diff
@@ -4,6 +4,7 @@ const CategoryContract = artifacts.require("CategoryContract");
 const DeveloperPool = artifacts.require("DeveloperPool");
 const Sintrop = artifacts.require("Sintrop");
 const ProducerContract = artifacts.require("ProducerContract");
+const ResearcherContract = artifacts.require("ResearcherContract");
```

Comment:
Não entendi muito bem esse contrato `2_deploy_contracts.js`.

Por que entram os contratos de `ActivistContract`, `ProducerContract` e `ResearcherContract`, mas não entram os dos demais tipos de usuários como Advisor, Developer e Contributor?

Esses demais contratos eu deveria adicionar ao realizar a #93 ?

Reply:
@pjvperes Essa issue #93  depende da issue que o André está trabalhando (que é esse PE aqui mesmo), pois vc vai utilizar esse [contrato](https://github.com/Sintrop/core-contracts/pull/104/files#diff-7e8deb26fd2d93ff478f4c505e33ffcfb15f30f4734ee548bbd1cb9000ae9878) pra realizar a configuração do allowedUser. 
@rav-andre Vc pode explicar melhor essa issue #93 ?

Reply:
Quanto a sua dúvida mesmo @pjvperes se os osuários já tiverem sido adicionados no sistema e não tiverem sido configurados no allowedCaller, vc pode realizar tal configuração iguais os demais que vc comentou

Reply:
Vou atualizar na issue algumas recomendações de desenvolvimento. Assim que esse PR for mergeado vc vai poder trabalhar na issue @pjvperes

Reply:
@pjvperes esse contrato migrations serve para configurar automaticamente ações nos nossos contratos depois de feito o deploy. Uma alternativa seria de usar o Remix para testar e fazer exatamente esses passos do migrations "na mão". Então sim está faltando ainda configurar a parte do contributors, advisors e developers. 
Separamos então que o @everson-ever vai seguir com a parte do developers e você faz essa configuração para contributors e advisors.